examples/metrics-basic/src/main.rs

```diff
@@ -24,24 +33,16 @@ fn init_meter_provider() -> opentelemetry_sdk::metrics::SdkMeterProvider {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+tracing_subscriber::registry()
```

Comment:
modified example just to demo the working on new reader. These changes will be reverted before merging, this is only to help anyone wanting to quickly test.

Reply:
this is removed. Pick the changes from the first commit if one wants to try it out.